indices/user.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <title>User list</title>
+    <link rel="stylesheet" , href="index.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <script src="index.js"></script>
+</head>
+
+<body>
+    <header onclick="hideLoginForm()">
```

Comment:
try to listen event on root document  instead of each part like `header, main content, footer...`.

```javascript
document.addEventListener('click', ......
````